themes/tdr/login/messages/messages_en.properties

```diff
@@ -20,7 +20,10 @@ loginTotpUnableToScan=I want to enter a key instead
 mainHeader=Transfer Digital Records
 passwordNew=New password
 passwordReset=Password reset
-passwordRestrictions=Password must be at least 10 characters
+passwordRestrictions=Password must be at least 8 characters
+invalidPasswordMinLengthMessage=Invalid password: Your password must contain a minimum of {0} characters
+invalidPasswordMinUpperCaseCharsMessage=Invalid password: Your password must include at least {0} uppercase character
+invalidPasswordMinDigitsMessage=Invalid password: Your password must include at least {0} number
```

Comment:
Should "number" be plural, ie "numbers", or "number(s)"?

Reply:
In the final design it is shown as just "number"

Reply:
OK. Fair enough